lib/clients/reports.ex

```diff
@@ -2,12 +2,24 @@ defmodule GlassFactoryApi.Clients.Reports do
   alias GlassFactoryApi.ApiClient
   alias GlassFactoryApi.Clients.Reports.TimeReport
 
+  @moduledoc """
+  Provides function that get client reports from GlassFactory
+  """
+
+  @type parameters() :: [
```

Comment:
This is the created type @britto, WDTY? 

Refers to [this comment](https://github.com/Cendrao/glass_factory_api/pull/36#discussion_r351435967).

Reply:
Looks good! ✨

Are the types correct? By looking at the examples it seems that we expect a `Date` rather than a `String` in the `start` field.

Reply:
Oops~ 😅 

Fixed!